app/common/src/popup/Popup.js

```diff
@@ -2,42 +2,29 @@
 
 import * as React from 'react';
 import { View, StyleSheet } from 'react-native';
-import { Button } from '@kiwicom/react-native-app-common';
 
 import Modal from '../Modal';
 
 type Props = {|
   children: React.Node,
   isVisible: boolean,
   onClose: () => void,
-  onSave: () => void,
+  style?: Object,
 |};
 
 export const POPUP_PADDING = 20;
 
 export default class Popup extends React.Component<Props> {
   onClose = () => this.props.onClose();
 
-  onSave = () => {
-    this.props.onSave();
-    this.onClose();
-  };
-
   render = () => (
     <Modal
       isVisible={this.props.isVisible}
       style={styles.modal}
       backdropOpacity={0.5}
       onBackdropPress={this.onClose}
     >
-      <View style={styles.content}>
-        {this.props.children}
-        <Button
-          title="Save"
-          styles={{ button: styles.button }}
-          onPress={this.onSave}
-        />
-      </View>
+      <View style={styles.body}>{this.props.children}</View>
```

Comment:
All popups should contain something like `ScrollView` otherwise it's unusable when content is too long...